backend/app/views/backend/shared/_index_items_search_and_filter.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="btn js-index-searcher-button">
+  <svg class="icon">
+    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-search"></use>
+  </svg>
+</div>
+<div class="btn">
+  <svg class="icon">
+    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-sort"></use>
+  </svg>
+</div>
+<div class="searcher js-index-searcher" <% if @search.present? %>style="display: block;"<% end %>>
```

Comment:
I think that, that would work? But I thought I tried and didn't. Another option could also be to have the magnifier icon become a "x" and if you click there it dismisses and clears the search?